public/index.html

```diff
@@ -23,7 +23,13 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-  <title>ForkDelta: The alternative EtherDelta trading interface</title>
+  <meta property="og:url" content="%PUBLIC_URL%" />
+  <meta property="og:title" content="ForkDelta | Decentralized Ethereum Token Exchange" />
+  <meta property="og:description" content="ForkDelta is a fully decentralized Ethereum Token Exchange with the most ERC20 listings of any exchange. Start trading cryptocurrency today." />
+  <meta property="og:image" content="%PUBLIC_URL%/images/logo.png" />
```

Comment:
One more correction: there isn't an `/images/logo.png`. We could probably use https://github.com/forkdelta/next/blob/master/android-chrome-256x256.png for the logo, it would be `%PUBLIC_URL%/android-chrome-256x256.png`.

Sorry I didn't notice before.